assets/app/views/_triggers.html

```diff
@@ -3,9 +3,16 @@
     <span ng-switch="build.status" class="hide-ng-leave">
       <span ng-switch-when="Complete" class="fa fa-check text-success" aria-hidden="true" style="margin-right: 5px;"></span>
       <span ng-switch-when="Failed" class="fa fa-times text-danger" aria-hidden="true" style="margin-right: 5px;"></span>
+      <span ng-switch-when="Cancelled" class="fa fa-ban text-warning" aria-hidden="true"></span>
       <span ng-switch-default class="fa fa-refresh fa-spin" aria-hidden="true" style="margin-right: 5px;"></span>
     </span>
-    <span>A build of {{build.metadata.labels.buildconfig}} is {{build.status | lowercase}}.</span>
-    <span ng-if="build.status != 'Complete' && build.status != 'Failed' && trigger.imageChangeParams.automatic">  A new deployment will be created automatically once the build completes.</span>
+    <span>
+      A build of {{build.metadata.labels.buildconfig}}
```

Comment:
probably need another switch
Failed: A build of ... failed
Cancelled: A build of ... was cancelled
Default: A build of ... is lower(status)